content/en/bits_ai/managing_incidents.md

```diff
@@ -1,46 +1,70 @@
 ---
 title: Managing Incidents
 kind: guide
-disable_toc: false
-private: true
-is_beta: true
+further_reading:
+- link: "bits_ai/"
+  tag: "Documentation"
+  text: "Bits AI Overview"
+- link: "bits_ai/getting_started"
+  tag: "Documentation"
+  text: "Getting Started"
+- link: "bits_ai/guide/query_examples"
+  tag: "Documentation"
+  text: "Example Natural Language Queries"
 ---
 
-{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLSfBuPfdyhgqjjduDYpOM5twJdkdDnTTxJdCCWonauaBxWTCnQ/viewform" >}}
-Bits AI is in private beta. Fill out this form to join the wait list.
-{{< /callout >}}
+## Overview
 
 <div class="alert alert-warning">The following features are part of the <a href="https://www.datadoghq.com/product/incident-management/">Datadog Incident Management</a> product.</div>
 
+Bits AI simplifies incident management processes, improves collaboration, and provides valuable support for incident responders, making it a beneficial tool for efficient incident resolution.
+
 ## Prerequisites
 
-- Make sure you've enabled the "Mirror all incident Slack channel messages to its incident timeline in real-time" in **[Incident > Settings > Integrations][3] > Slack**. This allows Datadog to generate summaries and postmortems from Slack conversations.
-- The incident channel name must be prefixed with `#incident-`. This enables you to use Bits AI in Slack (`@Datadog`) to ask questions about the incident.
-- Your Datadog account must be connected to Slack. Running a query automatically initiates this process, which can be completed by following the prompts.
+- Your Datadog account must be connected to Slack. Running the `/dd` connect command automatically initiates this process, which can be completed by following the prompts.
+- In **[Incident > Settings > Integrations][3] > Slack**, enable the **"Push Slack channel messages to the incident timeline"** and **"Activate Bits AI features in incident Slack channels for your organization"** toggles. This allows Datadog to ingest Slack conversations into the incident timeline to generate summaries and postmortems. **Note**: a Slack workspace can only have Bits AI activated for one Datadog Organization.   
+- Incident channels you'd like to work with Bits AI must be prefixed with `#incident-`. 
+- To ask Bits AI questions about incidents from any Slack channel, you must invite Bits AI to that channel. Run the `@Datadog` command and follow the instructions on the screen. 
+
+{{< img src="bits_ai/managing_incidents/integration-settings.png" alt="Slack integration settings in Datadog" style="width:90%;">}}
+
+## View incident summaries
+
+When you join an incident channel in Slack, you automatically receive a summary of the incident. The channel must be connected to Incident Management and have at least ten messages. The summary, which is only visible to you, does not persist across reloads, between desktop and mobile apps, or across sessions.
```

Comment:
Can we bold the part that says "at least ten messages"? It's a common issue that people run into. They try summarizing when there are only 1 or 2 messages and it fails because we can't generate a summary with that little content.

Reply:
```suggestion
When you join an incident channel in Slack, you automatically receive a summary of the incident. The channel must be connected to Incident Management and **have at least ten messages**. The summary, which is only visible to you, does not persist across reloads, between desktop and mobile apps, or across sessions.
```